index.js

```diff
@@ -82,12 +87,6 @@ module.exports = {
         ignoreAtRules: ['else'],
       },
     ],
-    'block-closing-brace-newline-after': [
-      'always',
-      {
-        ignoreAtRules: ['if', 'else'],
-      },
-    ],
```

Comment:
Deprecated in Stylelint v15, emits a warning message when linting is ran that was missed in beta testing.

Stylelint relies on Prettier rules to format now, so we go with Prettier's defaults.

> When we created these rules, pretty printers (like [Prettier](https://prettier.io/)) didn't exist. They now offer a better way to consistently format code, especially whitespace. Linters and pretty printers are complementary tools that work together to help you write consistent and error-free code.